src/submission/parseErrors/index.jsx

```diff
@@ -102,51 +97,61 @@ class ParseErrors extends Component {
               {props.pagination.total} {errorText} with Formatting Errors
             </h2>
           )}
-          <p className="usa-font-lead">
+          <p className='usa-font-lead'>
             The uploaded file is not formatted according to the requirements
             specified in the{' '}
             <a
-              rel="noopener noreferrer"
-              target="_blank"
-              href={(filingPeriod==='2018')?'https://s3.amazonaws.com/cfpb-hmda-public/prod/help/2018-hmda-fig-2018-hmda-rule.pdf':
-              `https://s3.amazonaws.com/cfpb-hmda-public/prod/help/${filingPeriod}-hmda-fig.pdf` }
+              rel='noopener noreferrer'
+              target='_blank'
+              href={
+                filingPeriod === '2018'
+                  ? 'https://s3.amazonaws.com/cfpb-hmda-public/prod/help/2018-hmda-fig-2018-hmda-rule.pdf'
+                  : `https://s3.amazonaws.com/cfpb-hmda-public/prod/help/${filingPeriod}-hmda-fig.pdf`
+              }
             >
               Filing Instructions Guide
             </a>{' '}
-            for data collected in {filingPeriod}{(filingPeriod==='2018')? ` incorporating the ${filingPeriod} HMDA Rule.`:'.' }
+            for data collected in {filingPeriod}
+            {filingPeriod === '2018'
+              ? ` incorporating the ${filingPeriod} HMDA Rule.`
+              : '.'}
           </p>
         </header>
         {renderTSErrors(props)}
         {renderLarErrors(props)}
-        <Pagination isFetching={props.isFetching} target="parseErrors" />
+        <Pagination isFetching={props.isFetching} target='parseErrors' />
         <RefileWarning />
       </section>
     )
   }
 }
 
 function errorResponseParser(errorResponse, uliNeeded) {
-  let apiErrorMessages =[]
-
-  errorResponse.forEach(function(errorsFound,i) {
-    if(errorsFound && errorsFound.errorMessages)
-    for (const [index, errorMessage] of errorsFound.errorMessages.entries()) {
-      let inputContent = errorMessage.inputValue
-
-      if (errorMessage.inputValue==='') inputContent = <em>(blank)</em>
- 
-      apiErrorMessages.push(
-        <tr key={`${i}${index}`}>
-          <td>{errorsFound.rowNumber}</td>
-          {uliNeeded ? <td>{errorsFound.estimatedULI}</td>:null}
-          <td>{errorMessage.fieldName}</td>
-          <td>{inputContent}</td>
-          <td> {errorMessage.validValues}</td>
-         </tr>
-      );
-    }
-})
-return apiErrorMessages
+  let apiErrorMessages = []
+
+  errorResponse.forEach(function(errorsFound, i) {
+    if (errorsFound && errorsFound.errorMessages)
+      errorResponse.forEach(function(errorsFound, i) {
+        if (errorsFound && errorsFound.errorMessages) {
```

Comment:
Looks like these two lines were doubled, there should only be one set of

```
errorResponse.forEach(function(errorsFound, i) {
    if (errorsFound && errorsFound.errorMessages) {
```

Reply:
🤕